### ISSUE TITLE
SpreadsheetApp.flush()

### ISSUE THREAD

**Issue by khalilchoudhry** [dan-kushnaryov/gas-mock-globals#24]
Hi again, recently I had to include [SpreadsheetApp.flush()](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet-app#flush) in my project & now almost all the tests are failing bcoz flush function is not available in ScriptAppStubConfiguration class.

Just wanted to discuss if a function like following would be fine:
`
flush(){
     return true;
}
`

We can't add the flush functionality obviously. Plz let me know, ill open a PR with this update.

**Comment by khalilchoudhry** [comment]
@jsarafajr @maxidr

**Comment by jsarafajr** [comment]
Hi @khalilchoudhry, sounds good.

Should it return boolean though? Isn't flush just "void"?

**Comment by khalilchoudhry** [comment]
yes, its just void. should we keep it empty? looks weird, that's all

**Comment by jsarafajr** [comment]
yeah, I'd leave it empty
